swing/src/net/sf/openrocket/gui/adaptors/DoubleModel.java

```diff
@@ -773,6 +773,14 @@ public double getValue() {
 	 */
 	public void setValue(double v) {
 		checkState(true);
+
+		if (v > maxValue) {
+			log.debug("Clipping value " + v + " to maximum " + maxValue + " for " + this);
+			v = maxValue;
+		} else if (v < minValue) {
+			log.debug("Clipping value " + v + " to minimum " + minValue + " for " + this);
+			v = minValue;
+		}
```

Comment:
Note, there is already a `MathUtil.clamp(double x, double min, double max)` method that does this (minus the logging - though that could be added with a check on whether the value has changed.

Reply:
This change seems extremely reasonable (and correct to me - to clamp the value). It would be *incorrect* to be out of the bounds of the model itself. Naturally, @SiboVG concern is around ensuring that there is proper testing as other components need to be validated that the bounds are specified correctly - which will require a fair amount more effort on the review than what I have this evening (though I am doing this).

Reply:
> Note, there is already a `MathUtil.clamp(double x, double min, double max)` method that does this (minus the logging - though that could be added with a check on whether the value has changed.

Yes, I wrote it explicitly to have the logging message, but I can rewrite it.